requirements.txt

```diff
@@ -2,6 +2,7 @@ algoliasearch>=2.0,<3.0
 flake8==3.7.7
 Flask-Limiter==1.0.1
 Flask-Migrate==2.5.2
+names==0.3.0
```

Comment:
Although this looks like a cool little project, I don't think it's necessary for us to pull in another dependency. Can we just generate random strings using something like this?

```python
import string
import random

name = ''.join(random.choices(string.ascii_uppercase + string.digits, k=10))
```
If you want it to be longer or shorter than 10 characters, you can change that last number to whatever length you want. If we're not testing based on numbers, this would be even shorter:
```python
''.join(random.choices(string.ascii_uppercase, k=10))
```